mobile-widgets/qml/Settings.qml

```diff
@@ -635,6 +635,30 @@ TemplatePage {
 						rootItem.settingsChanged()
 					}
 				}
+				TemplateLabel {
+					text: qsTr("GFLow")
+				}
+				TemplateTextField {
+					id: gfLow
+					text: PrefTechnicalDetails.gflow
+					inputMask: "99"
+					onEditingFinished: {
+						PrefTechnicalDetails.gflow = gfLow.text
+						rootItem.settingsChanged()
```

Comment:
OK, that looks right. I like the use of the `inputMask` -- there's so much more we could do to make the QML UI better...